File: src/entities/Services.kt
package com.example.myapplication.model

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

@Parcelize
data class Services(
    val name: String,
    val image: String,
    val description: String,
    val highlights: List<Highlight>,
    val plans: List<Plan>
) : Parcelable

File: src/entities/Services.ts
import {Column, Entity, PrimaryGeneratedColumn} from "typeorm";

@Entity()
export class Services{

    @PrimaryGeneratedColumn("rowid")
    id: number

    @Column({type:"text"})
    description: string

    @Column({type : "text"})
    image: string

    @Column({type : "text"})
    name: string
}